src/pages/item/item.ts

```diff
@@ -30,7 +37,22 @@ export class ItemPage {
   }
 
   ionViewDidLoad() {
-    console.log('ionViewDidLoad ItemPage');
+    this.afAuth.authState.subscribe(data =>{
+if(data.email && data.uid){
+      this.toast.create({
+        message : 'Welcome to UC-Market Place,  ' + data.email,
+        
+        duration:3000
+      }).present();
+    }else{
```

Comment:
Line up that curly bracket with the if statement please

Reply:
Actually it's more like that if statement should be indented to match the rest of it